### ISSUE TITLE
CSS: Portfolio Workshop Courseware Header should have a darker grey background

### ISSUE THREAD

**Issue by rediris** [gymnasium/tracker#186]
* as a fallback for if no course image is loaded to colorize the Courseware Header

<img width="843" alt="Screen Shot 2021-09-08 at 2 55 53 PM" src="https://user-images.githubusercontent.com/1010395/132568531-94cdd8d8-df40-43b4-8f09-116c99306717.png">
